src/modules/client/components/ClientProfileCard.tsx

```diff
@@ -368,26 +369,24 @@ const ClientProfileCard: React.FC<Props> = ({ client, onlyCard = false }) => {
                     : {}),
                 }}
               />
-              <Box sx={{ flexGrow: 1 }}>
+              <Stack sx={{ flexGrow: 1 }} gap={1} mt={2}>
```

Comment:
Just FYI this change was due to the fact that I originally put the delete button here. I decided to leave the change and not revert back since it seems to look fine this way and will handle adding more items to this stack if we ever need to.